deploy/olm-catalog/service-telemetry-operator/service-telemetry-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: service-telemetry-operator.v1.1.0
+- currentCSV: service-telemetry-operator.v1.0.4
```

Comment:
Why aren't you targetting this for 1.1.0? You'll cause a regression in the manifests if you don't add it to both. Since 1.1.0 isn't released (has some dashboarding work) I think you should just target that.

Reply:
TBH, I don't care. Happy to bump this to 1.1.0.